### PR TITLE
Nikis stannage docs

### PR DESCRIPTION
**Pull request by nikishe**

### CODE REVIEW

**Comment by willfurnass** [comment]
@nikishe Looks like this needs rebasing to resolve some conflicts with `master`.  Also, if this is ready for review then tag some reviewers.

**Review by Carldkennedy — APPROVED** [review]
LGTM

File: stanagepilot/software/libs/udunits.rst [hunk 1]
```diff
@@ -46,13 +46,14 @@ udunits was installed using Easybuild 4.7.0, build details can be found in ``$EB
...
 
-    1. load Module
 
-    2. Run “udunits2“
+1. load Module
+
+2. Run “udunits2“
 
-    3. For this test we convert 5km into miles, which produces the following results:
+3. For this test we convert 5km into miles, which produces the following results:
```
Comment (Carldkennedy) [inline stanagepilot/software/libs/udunits.rst:55]:
```suggestion


```

File: stanagepilot/software/libs/udunits.rst [hunk 2]
```diff
@@ -46,13 +46,15 @@ udunits was installed using Easybuild 4.7.0, build details can be found in ``$EB
 Testing
 -------
 
-    1. load Module
 
-    2. Run “udunits2“
+1. load Module
```
Comment (jkwmoore) [inline stanagepilot/software/libs/udunits.rst:50]:
```suggestion
1. Load module.
```

File: stanagepilot/software/libs/udunits.rst [hunk 3]
```diff
@@ -46,13 +46,15 @@ udunits was installed using Easybuild 4.7.0, build details can be found in ``$EB
...
+
+2. Run “udunits2“
```
Comment (jkwmoore) [inline stanagepilot/software/libs/udunits.rst:52]:
```suggestion
2. Run “udunits2“.
```

**Comment by jkwmoore** [comment]
Some slight edits, but otherwise LGTM.

Was this PR meant to have a lot more stuff in it? (Or did a bunch of it get cherry picked and merged in another PR?)

**Comment by nikishe** [comment]
@jkwmoore Some were in other PRs

File: stanagepilot/software/libs/udunits.rst [hunk 4]
```diff
@@ -43,16 +43,20 @@ This section is primarily for administrators of the system.
...
 
+------------------
+
 Testing
 -------
 
-    1. load Module
 
-    2. Run “udunits2“
+1. load module
```
Comment (jkwmoore) [inline stanagepilot/software/libs/udunits.rst:52]:
```suggestion
1. Load module.
```

**Review by jkwmoore — APPROVED** [review]

**Pull request merged**